src/Suave/Suave.fsproj

```diff
@@ -140,15 +179,7 @@
     <Reference Include="System.Web.Extensions" />
     <Reference Include="System.Xml" />
   </ItemGroup>
-  <Choose>
-    <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v4.5'">
-      <ItemGroup>
-        <Reference Include="FSharp.Core">
-          <HintPath>..\..\packages\FSharp.Core\lib\net40\FSharp.Core.dll</HintPath>
-          <Private>True</Private>
-          <Paket>True</Paket>
-        </Reference>
-      </ItemGroup>
-    </When>
-  </Choose>
+  <Target Name="BeforeBuild" BeforeTargets="Build" Condition="'$(TargetPlatform)' == 'coreclr'">
+    <Exec Command="..\..\tools\nuget\NuGet.exe restore -Source https://www.myget.org/F/dotnet-core;https://www.myget.org/F/fsharp-daily;https://nuget.org/api/v2 project.json -PackagesDirectory ..\..\packages -Verbosity detailed" />

```

Comment:
I don't know why the choose block was removed.  I don't recall doing it, I will try to come up with a better answer.

However, the new stuff is necessary for now.  
And this is what it does:
* It causes nugget to look at the project.json file.
* nugget then constructs a project.lock.json file which conveniently for me has partial paths to the physical files in the cache for all of the target platforms for compile and runtime dependencies.  
* These runtime dependencies may be specified in the runtimes.json file which is embedded in the nuget package.  It was done this way so that a developer doesnt need to download all of the platform dependencies for platforms that are not targeted.

You can see the project.lock.json file is parsed by the deploy,fsx script which copies the specified files to the deployment\platform\ directory.

Inorder to revert to packet it would be handy if packet would create this info in the lock file too, as well as parse and understand the embedded runtime.json file in the nuget packages.  Obviously the deploy package would need to be modified to parse the resultant packet.lock file.

I hope this helps

Kevin